scissors/main.c

```diff
@@ -0,0 +1,86 @@
+#include <stdio.h>
+#include <time.h>
+#include <stdlib.h>
+#include <stdio.h>
+
+enum {
+	NONE	= -1,
+	ROCK	= 0,
+	SCISSORS= 1,
+	PAPER	= 2,
+	OPTS_MAX
+}options;
```

Comment:
You do not need the variable `options`. Remove.

Reply:
Will fix.

Reply:
Fixed.